starter-code/iteration2/src/app/comments-component/comments-component.component.html

```diff
@@ -0,0 +1,17 @@
+<h3>Comments</h3>
```

Comment:
don't need to name the component, "component" as it will be added after automatic.